protos/ensembl/production/metadata/grpc/ensembl_metadata.proto

```diff
@@ -111,7 +111,10 @@ message AssemblyInfo {
   string level = 3;
   string name = 4;
   uint32 chromosomal = 5;
-  uint32 length = 6;
+  // length will be a string, because some browser can't handle a full 64bit value as a number,
+  // so the only way to ensure they can work for those clients is to handle them as strings
+  // source: https://github.com/protocolbuffers/protobuf/issues/2679
```

Comment:
I'm confused... Where will it be a string? In a tool to inspect grpc responses, such as Insomnia? I think web metadata api can still send it as number. It seems that [javascript number type has 53 bit of precision](https://stackoverflow.com/a/858857/3925302)

Reply:
> In a tool to inspect grpc responses, such as Insomnia?

Yes I added that comment after I noticed it was returning a string and tests were failing